packages/modular-scripts/src/esbuild-scripts/start/index.ts

```diff
@@ -200,10 +199,7 @@ class DevServer {
   });
 
   private runEsbuild = async (watch: boolean) => {
-    const plugins: esbuild.Plugin[] = [
-      svgrPlugin(),
-      incrementalReporterPlugin(),
-    ];
+    const plugins: esbuild.Plugin[] = [incrementalReporterPlugin()];
```

Comment:
why are we removing the `svgrPlugin` here?

Reply:
Common plugins are setup in `createEsbuildConfig.ts`